addons/caches/functions/fnc_moduleInitCaches.sqf

```diff
@@ -0,0 +1,48 @@
+#include "script_component.hpp"
+/*
+    Author:
+        Bridg
+
+    Description:
+        Creates specified number of caches
+
+    Parameters:
+        0: The module object <OBJECT>
+
+    Return value:
+        Nothing
+*/
+
+(_this select 1) params ["_module"];
+
+if (!isServer) exitWith {};
+
+private _numberOfCaches = _module getVariable [QGVAR(cacheNumber), 0];
+private _cacheTypes = parseSimpleArray (_module getVariable [QGVAR(cachePoolString), ""]);
+private _contentTypes = parseSimpleArray (_module getVariable [QGVAR(cacheContentPoolString), ""]);
+private _area = _module getVariable ["objectarea", []];
+
+for "_i" from 1 to _numberOfCaches do {
+
+    // get a position
+    _area deleteAt 4;
```

Comment:
Is this doing anything?
Area array doesn't have a 5th (index 4) element

Reply:
When I was getting this to work there was another element and it then wouldn't accept it as a valid array for the future use to had to delete the element.

Reply:
![image](https://user-images.githubusercontent.com/22099637/129688837-77ac80c8-aeba-4a8a-aca7-7ebd2196f56e.png)
5 elements